src/main.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 Felix Zailskas <felixzailskas@gmail.com>
+
+from demos import bdc_demo, db_demo, evp_demo, pipeline_demo
+
+if __name__ == "__main__":
+    while True:
+        try:
+            choice = int(input("(1) BDC\n(2) EVP\n(3) DB\n(4) Pipeline\n"))
+            match choice:
+                case 1:
+                    bdc_demo()
+                case 2:
+                    evp_demo()
+                case 3:
+                    db_demo()
```

Comment:
When db_demo() is selected before bdc_demo(), the app crashes. 
![app_crush](https://github.com/amosproj/amos2023ws06-sales-lead-qualifier/assets/45459787/cb0d6954-497f-4a56-8278-2bfbe5bf2c00)

Reply:
Addressed